docs/_pages/eventmonitoring.md

```diff
@@ -52,47 +53,40 @@ Or...
 subject.Should().NotRaise("SomeOtherEvent");
 ```
 
-There's also a generic version of `Monitor()`.
-It is used to limit which events you want to listen to.
-You do that by providing a type which defines the events.
+`Monitor()` is a generic method, but you will usually have the compiler infer the type. You _can_ specify an explicit type to limit which events you want to listen to:
 
 ```csharp
 var subject = new ClassWithManyEvents();
-using (var monitor = subject.Monitor<IInterfaceWithFewEvents>();
-{
-    
-}
+using var monitor = subject.Monitor<IInterfaceWithFewEvents>()
 ```
 
-This generic version of `Monitor()` is also very useful if you wish to monitor events of a dynamically generated class using `System.Reflection.Emit`.
-Since events are dynamically generated and are not present in parent class non-generic version of `Monitor()` will not find the events.
-This way you can tell the event monitor which interface was implemented in the generated class.
+This generic version of `Monitor()` is also very useful if you wish to monitor events of a dynamically generated class using `System.Reflection.Emit`. Since events are dynamically generated and are not present in parent class non-generic version of `Monitor()` will not find the events. This way you can tell the event monitor which interface was implemented in the generated class.
 
 ```csharp
 POCOClass subject = EmitViewModelFromPOCOClass();
 
 using (var monitor = subject.Monitor<ISomeInterface>())
```

Comment:
```suggestion
using var monitor = subject.Monitor<ISomeInterface>();
```

Reply:
Didn't want to use that in the docs since not everybody may be on C# 8 yet.

Reply:
Then I suggest we keep a full `using` scope everywhere in the docs?

```c#
using (var monitor = ...)
{
}
```

Reply:
On second hand, it might be fine to remind people of new syntax options like that.